tests/homework_session_7.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+from selenium import webdriver
+import time
+
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webelement import WebElement
+
+from tests import CHROME_PATH
+
+
+class OrangeHRMUserSearch(unittest.TestCase):
+
+    def setUp(self) -> None:
+        driver = webdriver.Chrome()
+        driver.maximize_window()
+        self.driver = driver
+        driver.get("http://hrm-online.portnov.com/")
+
+    def tearDown(self) -> None:
+        self.driver.quit()
+
+    def login(self, username, password):
+        self.driver.find_element_by_id('txtUsername').send_keys(username) if username else None
+
+        if password:
+            self.driver.find_element_by_id('txtPassword').send_keys(password)
+
+        self.driver.find_element_by_id('btnLogin').click()
+
+    def test_user_by_emp_id(self):
+        driver = self.driver
+        self.login("admin", "password")
+        time.sleep(1)
+        driver.find_element_by_id("empsearch_id").send_keys("319659")
+        driver.find_element_by_id("searchBtn").click()
+        result = driver.find_elements_by_xpath("//table/tbody/tr")
+        for item in result:
```

Comment:
When searching by ID you should only get one result back, in fact later in your script you have an assertion specifically checking that. Therefore there is no need for a loop.